src/early_multicellular_stage/editor/action_data/CellReproductionOrderActionData.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+/// <summary>
+///   Stores information needed to change the reproduction order of cells in an <see cref="EarlyMulticellularSpecies"/>.
+/// </summary>
+[JSONAlwaysDynamicType]
+public class CellReproductionOrderActionData : EditorCombinableActionData<EarlyMulticellularSpecies>
+{
+    public readonly int OldIndex;
+    public readonly int NewIndex;
+
+    public CellReproductionOrderActionData(int oldIndex, int newIndex)
+    {
+        OldIndex = oldIndex;
+        NewIndex = newIndex;
+    }
+
+    protected override ActionInterferenceMode GetInterferenceModeWithGuaranteed(CombinableActionData other)
+    {
+        return ActionInterferenceMode.NoInterference;
```

Comment:
Should this action combine with later reproduction order actions? Similarly to how multiple organelle moves in a row will combine. For example if someone moves the last cell to be the second cell and then moves the second cell to first, that could combine into one reproduction order action, right?